src/Estimators/tests/test_MomentumDistribution.cpp

```diff
@@ -189,7 +189,13 @@ TEST_CASE("MomentumDistribution::accumulate", "[estimators]")
   //std::cout<<"}\n\n\n";
 
   for (size_t id = 0; id < ref_data.size(); ++id)
-    CHECK(data[id] == Approx(ref_data[id]));
+  {
+     #ifdef MIXED_PRECISION
+     CHECK(data[id] == Approx(ref_data[id]).epsilon(0.00002));
+     #else
+     CHECK(data[id] == Approx(ref_data[id]));
```

Comment:
Can you explicitly put the default numerical comparison/epsilon for the full precisision case (currently line 196) to make it clear what the difference in the comparison is?